opm/autodiff/ParallelOverlappingILU0.hpp

```diff
@@ -110,30 +249,35 @@ class ParallelOverlappingILU0
       \param A The matrix to operate on.
       \param w The relaxation factor.
     */
-    ParallelOverlappingILU0 (const Matrix& A, const ParallelInfo& comm,
-                             field_type w)
-        : ilu_(A), comm_(comm), w_(w)
+    ParallelOverlappingILU0 (const Matrix& A, const int iluIteration, const field_type w )
```

Comment:
Ja, it's the n from ILUn. I never understood why this needed to be in a separate class. The 0 in the class name can be removed, but then again does the n version work in parallel. I don't think so.